tests/test_sliding_patch_wsi_dataset.py

```diff
@@ -137,12 +137,12 @@
 TEST_CASE_LARGE_0 = [
     {"data": [{"image": FILE_PATH, "level": 8, "size": (64, 50)}]},
     [
-        {"step_loc": (0, 0), "size": (64, 50), "level": 8, "ratios": (32914 / 128, 46000 / 179)},
-        {"step_loc": (0, 1), "size": (64, 50), "level": 8, "ratios": (32914 / 128, 46000 / 179)},
-        {"step_loc": (0, 2), "size": (64, 50), "level": 8, "ratios": (32914 / 128, 46000 / 179)},
-        {"step_loc": (1, 0), "size": (64, 50), "level": 8, "ratios": (32914 / 128, 46000 / 179)},
-        {"step_loc": (1, 1), "size": (64, 50), "level": 8, "ratios": (32914 / 128, 46000 / 179)},
-        {"step_loc": (1, 2), "size": (64, 50), "level": 8, "ratios": (32914 / 128, 46000 / 179)},
+        {"step_loc": (0, 0), "size": (64, 50), "level": 8, "ratio": 257.06195068359375},
```

Comment:
Actually, they are important changes. Now the downsampling is being read from the resolution metadata of whole slide images rather than using the image shape to infer that, so there will be a slight difference between them.